src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_params.py

```diff
@@ -92,6 +99,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config update', 'java_version', help="The version used to run your web app if using Java, e.g., '1.7' for Java 7, '1.8' for Java 8")
 register_cli_argument('appservice web config update', 'java_container', help="The java container, e.g., Tomcat, Jetty")
 register_cli_argument('appservice web config update', 'java_container_version', help="The version of the java container, e.g., '8.0.23' for Tomcat")
+register_cli_argument('appservice web config update', 'app_command_line', help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
```

Comment:
This is the property name exposed from the swagger. So by default, we expose the same from command line.
Since I thought about the same thing, let us break out the parity from xplat, and use the `--startup-file`